.github/styles/cloudflare/BrandNames.yml

```
@@ -0,0 +1,14 @@
+---
+# Warning: cloudflare.BrandNames
+#
+# Suggests correct spelling and capitalization for brand names.
+#
+# For a list of all options, see https://vale.sh/docs/topics/styles/
+
+extends: substitution
+message: "**Warning**: Use correct brand name: '%s' instead of '%s'."
+level: warning
+ignorecase: false
+swap:
+  CloudFlare: Cloudflare
```

To catch anyone missing the leading capital too :)
```suggestion
  (?!Cloudflare)([cC]loud[fF]lare): Cloudflare
```